services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email=None,
+                first_name=None,
+                last_name=None
+                ) -> User:
+    user = get_user_model().objects.create_user(username=username)

```

Comment:
You are calling `create_user` method with only username and then setting password separately. `create_user` method can also accept password as a parameter. It's better to pass password directly to `create_user` method to avoid calling `save` method multiple times. This is a code quality issue. Here's how you can do it:

python
user = get_user_model().objects.create_user(username=username, password=password)
```